app/assets/stylesheets/restaurants.scss

```diff
@@ -1,3 +1,112 @@
-// Place all the styles related to the Restaurants controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+@mixin make-float-boxes(){
+  float: left;
+  display: inline-block;
+}
+
+@mixin image-no-repeat($url){
+  background-image: url($url);
+  background-repeat: no-repeat;
+
+}
+.header{
+  position: relative;
+  width: 100%;
+  height: 200px;
+  background-size: cover;
+  @include image-no-repeat('header.png');
+
+  .header-text {
+    position: absolute;
+    width: 100%;
+    height: 50%;
+    top: 25%;
+    background-position: center;
+    @include image-no-repeat('header-text.png');
+  }
+
+}
+
+.list{
+
+  .filters-box{
+    position: relative;
+    width: 100%;
+    height: 150px;
+    background-color: cornflowerblue;
+
+    .filters-wrapper{
+      position: relative;
+      width:75%;
+      height: 80%;
+      top: 20px;
+      margin: auto;
+    }
+    .filter-item{
+      position: relative;
+      width: 30%;
+      height: 100px;
+      background-color: #2E2F30;
+      margin: 10px 10px 10px 10px;
+      @include make-float-boxes;
+    }
+  }
+
+  .restaurants-box{
+    position: relative;
+    width: 55%;
+    padding: 20px 0 0 20px;
+
+    .restaurant-item {
+      position: relative;
+      width: 90%;
+      height: 100px;
+      background-color: white;
+      margin: 10px 10px 10px 10px;
+      border-radius: 5px;
+
+      .restaurant-icon{
+        position: relative;
+        font-family: 'MyWebFont';
+        text-align: center;
+        font-size: 90px;
+        width: 15%;
+        height: 100%;
+        @include make-float-boxes;
+      }
+      .restaurant-wrapper {
+        position: relative;
+        width: 30%;
+        height: 50%;
+        top:25%;
+        @include make-float-boxes;
+
+        .restaurant-sm-wrapper{
+            position: relative;
+            width: 100%;
+            height: 50%;
+
+          .restaurant-name {
```

Comment:
Nested too deeply, while it's probably not a big deal, you might want a shallower approach or using top-level styles like `.headline`, `.sub_headline` etc